src/java.desktop/share/classes/java/applet/package-info.java

```diff
@@ -36,7 +36,11 @@
  * running applets. For example, the applet context could be a Web browser or an
  * applet development environment.
  * <p>
- * The APIs in this package are all deprecated without replacement.
+ * @deprecated.  This package has been deprecated and may be removed in
```

Comment:
Package declarations cannot have `@deprecated` tags, so `make docs` fails with this change. Also, since there is a tag here, the previous `<p>` is now empty and causes a warning. Both problems will be fixed by removing the `@deprecated` tag, while leaving the added text.